edgelet/contrib/config/linux/config.yaml

```diff
@@ -247,9 +247,38 @@ homedir: "/var/lib/iotedge"
 #
 # uri - configures the uri for the container runtime.
 # network - configures the network on which the containers will be created.
-#
+# 
+# Additional container network configuration such as enabling IPv6 networking 
+# and providing the IPAM settings can be achieved by specifying the relevant 
+# configuration in the network settings.
+# 
+# network:
+#   name: "azure-iot-edge"
+#   ipv6: true
```

Comment:
@arsing, afaik docker doesn't support ipv6 on Windows yet.  Will you validate this yaml config section to ensure ipv6 should not set to true on Windows?

Reply:
I can also add a check for that in the daemon itself so that it fails when the customer sets IPv6 to 'true' in config.yaml. However, the customer wouldn't even be able to get moby to start up on the machine if they had IPv6 configured in the daemon.json file. So, maybe 'iotedge check' is a better place to add that?

Reply:
I assume if `daemon.json` does not have `ipv6: true` and config.yaml does have `network.ipv6: true`, then the create network API will fail, regardless of Linux or Windows?

If so, then don't add any check to iotedged. Let it fail.

Add a check to `iotedge check` that does two things:

1. If Windows and `daemon.json` has `ipv6: true`, report an error.

1. If `daemon.json` does not have `ipv6: true` but `config.yaml` has `network.ipv6: true`, report an error.

Should be in the "Configuration" section.

Reply:
Yep, that's correct. I'll make the 'iotedge check' separately in another pull-request - does that work?

Reply:
Would be good to do it in this same PR. You can base it on the two `container_engine_*` checks.

Reply:
Cool, will do

Reply:
Done, added the relevant checks